src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -393,8 +393,13 @@ TEST_F(CommandRequestImplTest,
   EXPECT_CALL(app_mngr_, CheckPolicyPermissions(_, _, _, _))
       .WillOnce(Return(mobile_apis::Result::INVALID_ENUM));
 
+  smart_objects::SmartObjectSPtr response =
+      new smart_objects::SmartObject(smart_objects::SmartType_Map);
```

Comment:
@AKalinich-Luxoft use `MakeShared` please

Reply:
@AByzhynar fixed in 6cbc168f93754d569ef91cef7320931abdae4f19